docs/README.md

```diff
@@ -77,74 +83,69 @@ Write Compute includes two kinds of operations, which can be performed on the va
 N.B.: Currently, write compute is only supported in conjunction with active-passive replication. Support for 
 active-active replication is under development. 
```

Comment:
Do you want to update this part? Although we found a bug last Friday, I think we, still onboarded a few PROD cases already. (Or if you feel like it can be updated after GA that's also ok)

Reply:
The bugs we find are further and further apart, and that is a good sign of progress, but I would prefer removing this disclaimer after all known issues are fixed.